src/main/resources/META-INF/mods.toml

```diff
@@ -1,8 +1,9 @@
 modLoader="javafml" #mandatory
-loaderVersion="[25,)" #mandatory (24 is current forge version)
+loaderVersion="[36,)" #mandatory
+license="LGPL-2.1"
 [[mods]] #mandatory
 modId="searchablechests" #mandatory
-version="1.15-1.5.4-2" #mandatory
+version="1.16-1.5.4" #mandatory
```

Comment:
Is the version number correct? Or should it be incremented?

Reply:
This is correct. If you could just run delete the old launch configurations and run `gradlew genEclipseRuns` to generate the new ones I'd be happy to merge. I'll also credit you in the changelog when I publish the new version, unless you'd rather I don't. Just let me know!

Reply:
Should the launch configurations be in the repo? They include absolute paths specific to my system, which makes me think any dev using Eclipse will have to generate the files for their system for them to work (I am using IntelliJ IDEA, not sure how the launch files work).
As for credit, you can credit me as "antbrl", thanks for asking!

Reply:
Actually that's an excellent point. Since I was mainly using git for myself to track versions I hadn't thought about the local file issue. Could you actually just go ahead and delete them in this branch and I'll merge it in?

Reply:
OK done. Also when creating the PR I couldn't choose to create a new branch, so I chose the 1.15 branch but really it should be 1.16.